README.asciidoc

```diff
@@ -1,2 +1,31 @@
-TODO
-Instructions how to set up and run the code.
+This code is used for robots participating in robot basketball competions. The competition rules are described here: https://ut-robotics.github.io/robot-basketball-rules/
```

Comment:
A note about the readme structure:

https://ut-robotics.github.io/picr22-home/team_documentation_requirements.html

This file might work as the software readme but the main readme needs to fulfil more conditions. Just so you know

Reply:
The main readme with game logic documentation etc. is in the documentation branch...